forch/faucet_state_collector.py

```diff
@@ -1131,10 +1137,6 @@ def process_port_expire(self, timestamp, name, port, mac, expired_vlan=None):
 
             port_attr = self._get_port_attributes(name, port)
             if port_attr and port_attr['type'] == 'access':
-                if self._placement_callback:
```

Comment:
Do we need to remove the call of the `_placement_callback` in `process_port_learn()`?